configurations/dev.ts

```diff
@@ -1,125 +1,139 @@
-import { PreEngagementConfig, Translations, Configuration, MapHelplineLanguage } from './types';
+import { PreEngagementConfig, Translations, Configuration, MapHelplineLanguage, PreEngagementFormMutations } from './types';
```

Comment:
Most changes in this file are linter related, you can ignore, the only relevant chunk is [this ](https://github.com/techmatters/webchat/pull/149/files#r1037438365).